parser/src/main/java/me/vertretungsplan/parser/LegionBoardParser.java

```diff
@@ -119,7 +120,12 @@ private JSONArray getTeachers() throws IOException, JSONException {
 	}
 
 	private JSONArray getJSONArray(String url) throws IOException, JSONException {
-		return new JSONArray(httpGet(url, "UTF-8"));
+		try {
+			return new JSONArray(httpGet(url, "UTF-8"));
+		} catch (HttpResponseException e) {
+			e.printStackTrace();
```

Comment:
Letting the function throwing the exception did not work for me because then the execution stopped.